### ISSUE TITLE
[FR] Visualize metrics across different runs epoch by epoch

### ISSUE THREAD

**Issue by TeemuSo** [mlflow/mlflow#7060]
### Willingness to contribute

No. I cannot contribute this feature at this time.

### Proposal Summary

In deeping learning its important to be able to visualize loss convergences epoch by epoch. Its also important to compare different runs losses, to know how convergence is between the models. 

This feature is not currently supported by MLFlow, but it should be straightforward, as we already have necessary information residing in the runs.

### Motivation

> #### What is the use case for this feature?

Compare different models loss convergence or other time related metric

> #### Why is this use case valuable to support for MLflow users in general?

Its basic protocol when comparing deep learning model training runs

> #### Why is this use case valuable to support for your project(s) or organization?

Its basic protocol when comparing deep learning model training runs

> #### Why is it currently difficult to achieve this use case?

There is no such feature

**Comment by CamilleMaurice** [comment]
Dear @TeemuSo 
If I understand correctly to can log the loss values epoch by epoch during training. Then you need to run twice the training in order to get 2 runs to compare. In MLFlow ui, you can select 2 runs, then clic "Compare", then browse down the page and under "Metric" you can clic "loss" or "training loss" or whatever you called it. Then, a plot appears and you can visually compare.
I hope this helps !

**Comment by harupy** [comment]
@TeemuSo As @CamilleMaurice commented, you can compare different runs:

https://user-images.githubusercontent.com/17039389/195896027-25d13d41-9c0a-46be-bcc3-c27ad060d91b.mov

Let me know if this satisfies your needs.

**Comment by harupy** [comment]
@TeemuSo Feel free to reopen the issue if the existing compare-runs feature doesn't satisfy your needs.
